definitions/divorce/json/AuthorisationCaseField.json

```diff
@@ -9301,5 +9301,1979 @@
     "CaseFieldID": "labelOutcomeofDN",
     "UserRole": "caseworker-divorce-superuser",
     "CRUD": "R"
+  },
```

Comment:
Why all the fields has CRU?

Reply:
it reflects the Field permission of `caseworker-divorce-courtadmin_beta`(as for AC 1)

Reply:
I thought  we copy  `caseworker-divorce-courtadmin_beta` but with read permissions excluding these fields that the LA really need to update